rbd/encryption.go

```diff
@@ -0,0 +1,134 @@
+// +build !octopus,!nautilus
+
+package rbd
+
+// #cgo LDFLAGS: -lrbd
+// /* force XSI-complaint strerror_r() */
+// #define _POSIX_C_SOURCE 200112L
+// #undef _GNU_SOURCE
+// #include <errno.h>
+// #include <stdlib.h>
+// #include <rados/librados.h>
+// #include <rbd/librbd.h>
+import "C"
+
+import (
+	"unsafe"
+)
+
+// formt is the encryption format type
+type format C.rbd_encryption_format_t
+
+// Possible values for format:
+// ENCRYPTION_FORMAT_LUKS1: LUKS v1
+// ENCRYPTION_FORMAT_LUKS2: LUKS v2
+const (
+	ENCRYPTION_FORMAT_LUKS1 = format(C.RBD_ENCRYPTION_FORMAT_LUKS1)
```

Comment:
What I meant was, that you agreed before with @phlogistonjohn and @nixpanic to change the constants to Go style like that:

```suggestion
	EncryptionFormatLUKS1 = format(C.RBD_ENCRYPTION_FORMAT_LUKS1)
```

Reply:
Ah now I understand your comment.  I thought the request was for removing the RBD part.  I made the change you requested and will update the PR in a few moments.